src/Payment/OrderLines.php

```diff
@@ -457,10 +457,12 @@ private function get_item_category($product)
 
         //if product has taxonomy associated, retrieve voucher cat from there.
         $catTerms = get_the_terms($product->get_id(), 'product_cat');
-        foreach ($catTerms as $term) {
-            $term_id = $term->term_id;
-            $metaVoucher = get_term_meta($term_id, '_mollie_voucher_category', true);
-            $category = $metaVoucher ?: $category;
+        if(is_array($catTerms)){
+            foreach ($catTerms as $term) {
+                $term_id = $term->term_id;
+                $metaVoucher = get_term_meta($term_id, '_mollie_voucher_category', true);
+                $category = $metaVoucher ?: $category;
```

Comment:
There's a single variable set in a `foreach` here. Is this because the array is known to contain only a single entry? Or in other words, we only care about the last one?
If that's the case, you can just `$term = reset( $catTerms )` (or `end( $catTerms )` for specifically getting the last element) and move on from there without the `foreach`